src/configure/utilities/webAppNodeVersionConverter.ts

```diff
@@ -0,0 +1,36 @@
+import { JSONPath } from 'jsonpath-plus';
+import * as vscode from 'vscode';
+import { ArmRestClient } from '../clients/azure/armRestClient';
+import { AzureSession } from "../model/models";
+import { Messages } from '../resources/messages';
+
+export async function webAppRuntimeNodeVersionConverter(nodeVersion: string, armUri: string, azureSession: AzureSession): Promise<string> {
+    if (nodeVersion.indexOf('|') >= 0){
+        nodeVersion = nodeVersion.split('|')[1];
+    }
+    if (nodeVersion === 'lts') {
+        let versions: string[];
+        await vscode.window.withProgress(
+            {
+                location: vscode.ProgressLocation.Notification,
+                title: Messages.GettingNodeVersion
+            },
+            async () => {
+                const resultSelector = "$.value[?(@.name === 'node')].properties.majorVersions[*].runtimeVersions";
+                const response = await new ArmRestClient(this.azureSession).fetchArmData(armUri, 'GET');
+                versions = JSONPath({ path: resultSelector, json: response, wrap: false, flatten: true });
+            }
+        );
+        let maxVersion = 0;
```

Comment:
Initialise to current lts version

Reply:
Here we are comparing the major version which is numeric starts with 0. Trying to find the lts version by selecting the maximum major version.

Reply:
Got it. I just wanted to handle the case when no match happens or an error ocuurs. In such case we can fall back to current lts.

Reply:
falling back to lts will result in setup task failure.